README.md

```diff
@@ -1 +1,72 @@
-# strong-error-handler
\ No newline at end of file
+# strong-error-handler
+
+Error handler for use in development (debug) and production environments.
+
+ - When run in production mode, error responses are purposely undetailed
```

Comment:
`error responses are not detailed by design`.

I missed this during review, @richardpringle feel free to submit a PR and assign it to me for review.